static/js/libs/fileValidation.js

```diff
@@ -56,19 +84,21 @@ function imageRestrictions(file, restrictions) {
       const width = image.naturalWidth;
       const height = image.naturalHeight;
 
+      file.meta = {
+        width: width,
+        height: height
+      };
+
       if (
-        (file.name.indexOf("banner.") === 0 &&
-          width === 1218 &&
-          height === 240) ||
-        (file.name.indexOf("banner-icon.") === 0 &&
-          width === 240 &&
-          height === 240)
+        restrictions.whitelist &&
+        imageWhitelistHandler(file, image, restrictions.whitelist)
       ) {
         resolve(file);
         return;
       }
 
       const aspectRatio = width / height;
+      file.meta = aspectRatio;
```

Comment:
Is this `meta` used anywhere?
It's first set to `{ width, height }`, here overridden by `aspectRatio`, but it doesn't appear anywhere else in the diff?

Reply:
Ah I think I might have been using it for something else. Don't think the meta is used for anything so removed it :+1: good catch.